datasets/snow_simplified_japanese_corpus/README.md

```diff
@@ -0,0 +1,196 @@
+  The original texts are from "small_parallel_enja: 50k En/Ja Parallel Corpus for Testing SMT Methods", which is a bilingual corpus for machine translation.
+
+- **SNOW T23:**  
+  An expansion corpus of 35,000 sentences rewritten in easy Japanese (simple Japanese vocabulary) based on SNOW T15.  
+  The original texts are from "Tanaka Corpus" (http://www.edrdg.org/wiki/index.php/Tanaka_Corpus).
+
+### Supported Tasks and Leaderboards
+
+It can be used for automatic text simplification in Japanese as well as translating simple Japanese into English and vice-versa.
+
+### Languages
+
+Japanese, simplified Japanese, and English.
+
+## Dataset Structure
+
+### Data Instances
+
+SNOW T15 is xlsx file with ID, "#日本語(原文)" (Japanese (original)), "#やさしい日本語" (simplified Japanese), "#英語(原文)" (English (original)).  
+SNOW T23 is xlsx file with ID, "#日本語(原文)" (Japanese (original)), "#やさしい日本語" (simplified Japanese), "#英語(原文)" (English (original)), and "#固有名詞" (proper noun).
+
+### Data Fields
+
+- `ID`: sentence ID.
+- `original_ja`: original Japanese sentence.
+- `simplified_ja`: simplified Japanese sentence.
+- `original_en`: original English sentence.
+- `proper_noun`: (included only in SNOW T23) Proper nowus that the workers has extracted as proper nouns. The authors instructed workers not to rewrite proper nouns, leaving the determination of proper nouns to the workers.
+
+### Data Splits
+
+[N/A]
+
+## Dataset Creation
+
+### Curation Rationale
+
+A dataset on the study of automatic conversion to simplified Japanese (Japanese simplification).
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+- **SNOW T15:**  
+  The original texts are from "small_parallel_enja: 50k En/Ja Parallel Corpus for Testing SMT Methods", which is a bilingual corpus for machine translation.
+
+- **SNOW T23:**  
+  The original texts are from "Tanaka Corpus" (http://www.edrdg.org/wiki/index.php/Tanaka_Corpus).
+
+#### Who are the source language producers?
+
+[N/A]
+
+### Annotations
+
+#### Annotation process
+
+- **SNOW T15:**  
+  Five students in the laboratory rewrote the original Japanese sentences to simplified Japanese all by hand.  
+  The core vocabulary is restricted to 2,000 words where it is selected by accounting for several factors such as meaning preservation, variation, simplicity and the UniDic word segmentation criterion.
+
+- **SNOW T23:**  
+  Seven people, gathered through crowdsourcing, rewrote all the sentences manually.  
+  Each worker rewrote 5,000 sentences, of which 100 sentences were rewritten to be common among the workers.  
+  The average length of the sentences was kept as close to the same as possible so that the amount of work was not varied among the workers.
+
+#### Who are the annotators?
+
+Five students for SNOW T15, seven crowd workers for SNOW T23.
+
+### Personal and Sensitive Information
+
+[N/A]
```

Comment:
```suggestion
[More Information Needed]
```

Can you replace all the [N/A] with [More Information Needed] (except maybe splits and language producers)

Reply:
done